src/QMCHamiltonians/NonLocalTOperator.cpp

```diff
@@ -44,7 +45,30 @@ bool NonLocalTOperator::put(xmlNodePtr cur)
   bool success = m_param.put(cur);
   plusFactor=Tau*Gamma;
   minusFactor=-Tau*(1.0-Alpha*(1.0+Gamma));
-  return use_tmove=="yes";
+  int v_tmove=TMOVE_OFF;
+  std::ostringstream o;
+  if(use_tmove=="no")
+  {
+    v_tmove=TMOVE_OFF;
+    o << "  Using Locality Approximation";
+  }
+  else if(use_tmove=="yes"||use_tmove=="v0")
+  {
+    v_tmove=TMOVE_V0;
+    o << "  Using Non-local T-moves v0, M. Casula, PRB 74, 161102(R) (2006)";
+  }
+  else if(use_tmove=="v1")
+  {
+    v_tmove=TMOVE_V1;
+    o << "  Using Non-local T-moves v1, M. Casula et al., JCP 132, 154113 (2010)";
+  }
+  else
+  {
+    APP_ABORT("NonLocalTOperator::put uknown nonlocalmove option");
```

Comment:
spelling of 'unknown'.   Also it would be nice to print the given option in the error message.